LICENSE.LESSER

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Why is this file named `LICENSE.LESSER`? Even in the case of the LGPL I believe it's meant to just be `LICENSE`